ios/Capacitor/Capacitor/CAPPlugin.m

```diff
@@ -1,5 +1,6 @@
 #import "CAPPlugin.h"
 #import "CAPBridgedJSTypes.h"
+#import "CAPInstanceConfiguration.h"
```

Comment:
```suggestion
```

Not needed, I think you imported it on the first commit when the function returned CAPInstanceConfiguration